codeforamerica/vita-min
# Fyst 789/add child credit xml pdf

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-789

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Adds the child care and dependent credit to the return xml and the pdf

## How to test?
- Automated tests provided
- To test manually
  - Use the john mfj 8 dependents user and create an idaho intake
  - Progress through the flow, ultimately clicking on download your return' to see the pdf
  - Observe that 2000 is the credit
  - Click 'view main xml'
  - Observe the new 'Form39R' element exists and has the correct value

## Screenshots (for visual changes)
- Before
  - XML
![image](https://github.com/user-attachments/assets/f2c4e8eb-cb4f-428c-ad8c-b2755d2098f9)

  - PDF
![image](https://github.com/user-attachments/assets/40e0445e-f7d8-47d8-975f-7f1079f5bef2)

- After
  - XML
![image](https://github.com/user-attachments/assets/ce36217b-e189-4971-b713-848092e31337)

  - PDF
![image](https://github.com/user-attachments/assets/3e69eacd-3005-43b7-acae-6ccde10b0411)
